votor/src/staked_validators_cache.rs

```diff
@@ -566,7 +558,7 @@ mod tests {
 
     #[test]
     fn test_alpenglow_port_override() {
-        agave_logger::setup();
+        solana_logger::setup();
```

Comment:
Same, you need to do solana_logger->agave_logger. But actually I believe we can remove all the logger setup.